liquidator/liquidator.js

```diff
@@ -125,14 +125,12 @@ class Liquidator {
     });
 
     // If an override is provided, use that price. Else, get the latest price from the price feed.
-    const price = liquidatorOverridePrice ? this.toBN(liquidatorOverridePrice) : this.priceFeed.getCurrentPrice();
+    const price = liquidatorOverridePrice
+      ? this.toBN(liquidatorOverridePrice.toString())
```

Comment:
unrelated change: I added this `.toString` as this would throw if the param was a number. this is bad behavior as it would crash the bot if set to a non-string.